src/loaders/jobs/taskfarmer/task_generator.py

```diff
@@ -42,13 +42,12 @@
 
 # estimated execution time (in minutes) for each tool to process a chunk of data
 TASK_META = {'checkm2': {'chunk_size': 5000, 'exe_time': 60},
-             'gtdb_tk': {'chunk_size': 1000, 'exe_time': 90}}
-NODE_TIME_LIMIT = 10  # hours
+             'gtdb_tk': {'chunk_size': 1000, 'exe_time': 65}}
+NODE_TIME_LIMIT = 5  # hours  # TODO: automatically calculate this based on tool execution time and NODE_THREADS
```

Comment:
per benchmark, each task will take ~50 minutes. Manually set exe_time and NODE_TIME_LIMIT so that we reserve just enough nodes with 4 tasks per node. More configurable settings will be done at a later date.

Reply:
5 * 300 / 65 = 4.6 batches of 4 1000 genome gtdb_tk runs. Why 4.6?

Reply:
The logic to compute task per node is` 5 * 60 // 65 = 4`. I did 5 hrs to give it a little bit more buffer.

Reply:
no idea where that 300 came from, 4.6 is 5 * 60 / 65...

Ok, so you're assuming 4 batches per node, so 4 * 65 = 260mins, so a 40min buffer. Can you document that?

Reply:
Just somewhere in here document the math, e.g. we're using 128/256 cpus as that gave us the best run time w/ 4 simultaneous instances of 1000 genomes running for 65 mins (w/ buffer), we're assuming 4 sequential runs, etc. When the script is changed to specify all this stuff on a tool by tool basis we can move the docs to the specific tool settings

Reply:
👍